packages/next/next-server/server/next-server.ts

```diff
@@ -239,10 +257,17 @@ export default class Server {
     res.setHeader('Cache-Control', 'public, max-age=31536000, immutable')
   }
 
+  protected getCustomRoutes() {
+    this.customRoutes = require(join(this.distDir, ROUTES_MANIFEST))
```

Comment:
nit: can we not have the side effect here?
```suggestion
    return require(join(this.distDir, ROUTES_MANIFEST))
```